src/sparseml/exporters/transforms/gemm_to_matmulinteger_add_cast_mul.py

```diff
@@ -99,17 +92,18 @@ def transform(self, model: ModelProto) -> ModelProto:
             ):
                 # we do not currently handle Gemms with transposed A
                 # or scalar multiples
-                _LOGGER.debug(f"Skipping due to unsupported attributes {attr}")
                 continue
+            _LOGGER.debug(f"Matched {match}")
+            self._transform_match(graph, model, match, attr)
```

Comment:
the worry about not refreshing the graph on every transform is the internal dicts that track connections may become stale and lead to bugs - if you are confident this won't happen in this case should comment, otherwise let's revert in both cases here

building the `ONNXGraph` is relatively cheap (O(n)), what it speeds up the most is nested node/init name lookups (used to go 3-5 layers deep :|)

Reply:
Yeah that's a fair point - should we also not deferred the node addition/removal for the same reason then?

Reply:
Hmm i think the only concern i would have for these is that the helper function this calls (add_quantized_conv_matmul_add_ops) adds nodes/initializers to the graph. But the transforms shouldn't consider those new additions. Ideally they would do the deferred add on the nodes, but they don't have access to the transform (could make accept an ONNXTransform object?)